PWGGA/GammaConv/AliAnalysisTaskEtaToPiPlPiMiGamma.h

```diff
@@ -182,7 +182,7 @@ class AliAnalysisTaskEtaToPiPlPiMiGamma: public AliAnalysisTaskSE
 		AliAnalysisTaskEtaToPiPlPiMiGamma( const AliAnalysisTaskEtaToPiPlPiMiGamma& ); // Not implemented
 		AliAnalysisTaskEtaToPiPlPiMiGamma& operator=( const AliAnalysisTaskEtaToPiPlPiMiGamma& ); // Not implemented
 
-        ClassDef( AliAnalysisTaskEtaToPiPlPiMiGamma, 6 );
+        ClassDef( AliAnalysisTaskEtaToPiPlPiMiGamma, 7 );
```

Comment:
the ClassDef must not be updated, since you do not change persistent member variables - please fix this as well

Reply:
ah right, thanks for explaining this!